modules/nf-core/seqkit/sliding/main.nf

```diff
@@ -37,4 +37,17 @@ process SEQKIT_SLIDING {
         seqkit: \$( seqkit | sed '3!d; s/Version: //' )
     END_VERSIONS
     """
+
+    stub:
+    prefix = task.ext.prefix ?: "${meta.id}"
+    if ("$fastx" ==~ /.+\.fasta$|.+\.fa$|.+\.fas$|.+\.fna$/) {
+        extension = "fasta"
+    }    """
```

Comment:
Can you move this onto the next line please.

Reply:
Done, thanks for having a look.